TTD/TTD.hpp

```diff
@@ -7,50 +9,40 @@ namespace TTD {
 	typedef struct Position {
 		unsigned __int64 Major; // Sequence
 		unsigned __int64 Minor; // Steps
-	} Position;
 
+		bool operator < (const Position& other)
+		{
+			if (this->Major == other.Major) 
+			{
+				return this->Minor < other.Minor;
+			}
+			else
+			{
+				return this->Major < other.Major;
+			}
+		}
+
+		bool operator > (const Position& other)
+		{
+			if (this->Major == other.Major)
+			{
+				return this->Minor > other.Minor;
+			}
+			else
+			{
+				return this->Major > other.Major;
+			}
+		}
+
+		bool operator == (const Position& other)
+		{
+			return this->Major == other.Major && this->Minor == other.Minor;
+		}
 
-	// Size: 0xA70
-	struct TTD_Replay_RegisterContext {
-		unsigned __int64 unk1;
-		unsigned __int64 unk2;
-		unsigned __int64 unk3;
-		unsigned __int64 unk4;
-		unsigned __int64 unk5;
-		unsigned __int64 unk6;
-		unsigned __int64 unk7;
-		USHORT cs;
-		USHORT ds;
-		USHORT es;
-		USHORT fs;
-		USHORT gs;
-		USHORT ss;
-		unsigned __int32 efl;
-		unsigned __int64 unk8;
-		unsigned __int64 unk9;
-		unsigned __int64 unk10;
-		unsigned __int64 unk11;
-		unsigned __int64 unk12;
-		unsigned __int64 unk13;
-		unsigned __int64 rax;
-		unsigned __int64 rcx;
-		unsigned __int64 rdx;
-		unsigned __int64 rbx;
-		unsigned __int64 rsp;
-		unsigned __int64 rbp;
-		unsigned __int64 rsi;
-		unsigned __int64 rdi;
-		unsigned __int64 r8;
-		unsigned __int64 r9;
-		unsigned __int64 r10;
-		unsigned __int64 r11;
-		unsigned __int64 r12;
-		unsigned __int64 r13;
-		unsigned __int64 r14;
-		unsigned __int64 r15;
-		unsigned __int64 rip;
-		USHORT fpcw;
-	};
+	} Position;
+
+	// MAX position use for computation
+	static const Position MAX = { (uint64_t)-1, (uint64_t)-1 };
```

Comment:
```suggestion
	static const Position POSITION_MAX = { (uint64_t)-1, (uint64_t)-1 };
```

To avoid conflict with other global names